tests/tape/test_jacobian_tape.py

```diff
@@ -55,6 +55,23 @@ def test_parameter_info(self, make_tape):
         }
 
 
+class TestTapeCopying:
```

Comment:
Might be worth adding an integration test similar to the original example in the #1345 issue to ensure that the results using the `jax` interface are correct.